modules/logging-enabling-loki-alerts.adoc

```diff
@@ -1,30 +1,37 @@
 // Module included in the following assemblies:
-// logging-5-7-configuration
+//
+// logging/logging_alerts/log-storage-alerts.adoc
 
 :_content-type: PROCEDURE
-[id="logging-loki-alerts_{context}"]
-= Enabling log based alerts with Loki
-Loki alerting rules use link:https://grafana.com/docs/loki/latest/logql/[LogQL] and follow link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules[Prometheus formatting]. You can set log based alerts by creating an `AlertingRule` custom resource (CR). `AlertingRule` CRs may be created for `application`, `audit`, or `infrastructure` tenants.
+[id="logging-enabling-loki-alerts_{context}"]
+= Enabling log-based alerts with Loki
```

Comment:
I think we need to find a better place for this documentation. The `log-based alerts` feature now available with Loki has no relation to the other "alerts" being talked about here:

- "log-based alerts" are custom alerts defined by the user using the AlertingRule custom resource for getting alerts for _their_ workloads based on log output
- "log storage alerts" like the Elasticsearch alerts mentioned below this section or the alerts defined by the Loki Operator (see an example [here](https://loki-operator.dev/docs/sop.md/)) are defined by our operators to alerts the cluster administrator about failures in the functions provided by the operators.

Reply:
Updated the structure to divide the docs into
- default log alerts (i.e. operator defined)
- custom logging alerts (user defined)

does this work better?